### PR TITLE
Table panel: Add alt and title text options to image cell type

### PR DESCRIPTION
**Pull request by codeincarnate**
**What is this feature?**

This PR adds the ability to edit alt text and title text in the image cell type of the table panel. This is important for accessibility and provides another avenue for surfacing data (via hovering with title text).

**Why do we need this feature?**

Improves accessibility and usability of the table panel image cell

**Who is this feature for?**

Users of the table image cell

Please check that:
- [x] It works as expected from a user's perspective.
- [x] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/plugins/panel/table/cells/ImageCellOptionsEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const onTitleChange = (e: FormEvent<HTMLInputElement>) => {
+    cellOptions.title = e.currentTarget.value;
+    onChange(cellOptions);
+  };
+
+  return (
+    <>
+      <Field
+        label="Alt text"
+        description="Alternative text that will be displayed if an image can't be displayed or for users that use a screen reader"
```
Comment (Clarity-89) [inline public/app/plugins/panel/table/cells/ImageCellOptionsEditor.tsx:23]:
```suggestion
        description="Alternative text that will be displayed if an image can't be shown or for users who use a screen reader"
```

File: public/app/plugins/panel/table/cells/ImageCellOptionsEditor.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      >
+        <Input onChange={onAltChange} defaultValue={cellOptions.alt} />
+      </Field>
+
+      <Field label="Title text" description="Text that will show when the image is hovered by a cursor">
```
Comment (Clarity-89) [inline public/app/plugins/panel/table/cells/ImageCellOptionsEditor.tsx:28]:
```suggestion
      <Field label="Title text" description="Text that will be displayed when the image is hovered over by a cursor">
```

**Review by Clarity-89 — APPROVED** [review]

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 3]
```diff
@@ -157,6 +157,12 @@ Toggle the **Apply to entire row** switch, to apply the background color that's
 
 Cells can be displayed as a graphical gauge, with several different presentation types.
 
+#### Image
+
+If you want to set the alternative text of an image, use the **Alt text** option to do so which will make alternative text avaiable for screen readers and cases when images can't be loaded.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:162]:
```suggestion
Use the **Alt text** option to set the alternative text of an image. The text will be available for screen readers and in cases when images can't be loaded.
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 4]
```diff
@@ -157,6 +157,12 @@ Toggle the **Apply to entire row** switch, to apply the background color that's
...
+
+Additionally, if you'd like to set the title text, use the **Title text** option to do so which will make the title text display when the image is hovered with a cursor.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:164]:
```suggestion
Use the **Title text** option to set the text that's displayed when the image is hovered over with a cursor.
```

File: docs/sources/panels-visualizations/visualizations/table/index.md [hunk 5]
```diff
@@ -157,6 +157,12 @@ Toggle the **Apply to entire row** switch, to apply the background color that's
 
 Cells can be displayed as a graphical gauge, with several different presentation types.
 
+#### Image
+
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/table/index.md:161]: Can you add a basic description of this cell type.

**Review by imatwawana — COMMENTED** [review]
Thanks for adding the doc update. I've added a comment and some suggestions.

**Comment by codeincarnate** [comment]
Thanks @imatwawana! I've made the suggested changes 😄

**Review by imatwawana — APPROVED** [review]
Ah somehow I didn't see the image description that was already there. Good stuff!

**Comment by codeincarnate** [comment]
No worries, I missed it too! 😅

**Pull request merged**